scripts/monitoring/run_monitor_containers.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python /home/ubuntu/evalai/scripts/monitoring/monitor_containers.py
```

Comment:
Can we please use the bash command to get the absolute path for this folder and then append it with the relative path to this file?

Reply:
@RishabhJain2018  done